libraries/api-extractor-model/src/model/ApiIndexSignature.ts

```diff
@@ -45,7 +47,7 @@ export interface IApiIndexSignatureOptions
  * @public
  */
 export class ApiIndexSignature extends ApiParameterListMixin(
-  ApiReleaseTagMixin(ApiReturnTypeMixin(ApiDeclaredItem))
+  ApiReleaseTagMixin(ApiReturnTypeMixin(ApiReadonlyMixin(ApiDeclaredItem)))
```

Comment:
@octogonz I think we might need to update the `ApiJsonSchemaVersion` as when older JSON files are loaded, the `isReadonly` value for index signatures will always be `false`.

Reply:
Agreed, adding a base class will add a new JSON field, and we increment the version when adding new fields. This is mainly for bookkeeping purposes, in case a tool wants to print a warning about the possibly incorrect default values.

Reply:
Done!